src/AmbientSounds.Uwp/Views/ShellPage.xaml

```diff
@@ -193,6 +193,13 @@
                                 Duration="0:0:0.5" />
                         </animations:Implicit.ShowAnimations>
                     </controls:SaveMixButton>
+                    <Button
+                        AutomationProperties.Name="{x:Bind strings:Resources.PlayRandom}"
+                        Click="{x:Bind ViewModel.PlayRandomSoundAsync}"
```

Comment:
After you add RelayCommand attribute, change this to PlayRandomSoundCommand

Reply:
Sorry forgot to mention, change this to `Command="{x:Bind ViewModel.PlayRandomSoundCommand}"`